### ISSUE TITLE
Cannot provide custom file matching pattern

### ISSUE THREAD

**Issue by darioielardi** [Julien-R44/japa-vscode#3]
When I run my tests from the terminal I can rely on the `files` param for example like this:
```ts
configure({
  ...processCliArgs(process.argv.slice(2)),
  ...{
    files: ['test/**/*.e2e-spec.ts'],
    ...
  },
});
```

But code lenses do not show up on those files because I can't set the same pattern to match for the extension.

**Comment by Julien-R44** [comment]
Thanks for the report 
For the moment, for simplicity, I have hardcoded the test file name pattern. 
https://github.com/Julien-R44/japa-vscode/blob/main/src/index.ts#L14
it should be quite easy to fix anyway. I'll try by the end of the week

**Comment by darioielardi** [comment]
Great, thanks for your work and the quick reply!

**Comment by Julien-R44** [comment]
So I added this in the configuration : 

> `tests.filePattern`: The glob pattern to use when searching for tests. Defaults to **/*.{test,spec}.{ts,js}

I wish it could be automatic. However it seems quite complicated, the only solution would be to use an AST with complicated operations, and with many limitations. 

I hope this will suit you! Feel free to re-open the issue if any problems

**Comment by Julien-R44** [comment]
Was published as v0.0.7 on the Marketplace !

**Comment by darioielardi** [comment]
Works great, thank you!
